components/user_settings/sidebar/index.js

```diff
@@ -30,6 +30,12 @@ function mapStateToProps(state) {
             'show_unread_section',
             (config.ExperimentalGroupUnreadChannels === GroupUnreadChannels.DEFAULT_ON).toString()
         ),
+        channelSwitcherOption: getPreference(
```

Comment:
Should it be a `getBoolPreferences` too?

Reply:
will make the changes

Reply:
will not 😄